packages/web-pkg/src/components/TextEditor.vue

```diff
@@ -71,7 +50,59 @@ export default defineComponent({
       )
     })
 
-    onMounted(() => {
+    const loadTranslations = () => {
+      return Promise.all([
+        import('@toast-ui/editor/dist/i18n/ar'),
+        import('@toast-ui/editor/dist/i18n/cs-cz'),
+        import('@toast-ui/editor/dist/i18n/de-de'),
+        import('@toast-ui/editor/dist/i18n/en-us'),
+        import('@toast-ui/editor/dist/i18n/es-es'),
+        import('@toast-ui/editor/dist/i18n/fi-fi'),
+        import('@toast-ui/editor/dist/i18n/fr-fr'),
+        import('@toast-ui/editor/dist/i18n/gl-es'),
+        import('@toast-ui/editor/dist/i18n/hr-hr'),
+        import('@toast-ui/editor/dist/i18n/it-it'),
+        import('@toast-ui/editor/dist/i18n/ja-jp'),
+        import('@toast-ui/editor/dist/i18n/ko-kr'),
+        import('@toast-ui/editor/dist/i18n/nb-no'),
+        import('@toast-ui/editor/dist/i18n/nl-nl'),
+        import('@toast-ui/editor/dist/i18n/pl-pl'),
+        import('@toast-ui/editor/dist/i18n/ru-ru'),
+        import('@toast-ui/editor/dist/i18n/sv-se'),
+        import('@toast-ui/editor/dist/i18n/tr-tr'),
+        import('@toast-ui/editor/dist/i18n/uk-ua'),
+        import('@toast-ui/editor/dist/i18n/zh-cn'),
+        import('@toast-ui/editor/dist/i18n/zh-tw')
+      ])
+    }
+
+    const loadSyntaxHighlighting = async () => {
+      const [plugin] = await Promise.all([
+        import(
+          `@toast-ui/editor-plugin-code-syntax-highlight/dist/toastui-editor-plugin-code-syntax-highlight-all.js`
+        ),
+        import(
+          // @ts-ignore
+          `@toast-ui/editor-plugin-code-syntax-highlight/dist/toastui-editor-plugin-code-syntax-highlight.css`
+        ),
+        // @ts-ignore
+        import('prismjs/themes/prism.css')
+      ])
+
+      return plugin.default
+    }
+
+    onMounted(async () => {
+      let codeSyntaxHighlight: unknown
+      if (!props.isReadOnly && unref(isMarkdown)) {
+        // only needed for md documents with write permissions
+        await loadTranslations()
+        codeSyntaxHighlight = await loadSyntaxHighlighting()
```

Comment:
Syntax highlighting should be possible even if read only ? 🤷
Check space readme for example

Reply:
Ah okay, I didn't know that 👍

Reply:
I think that also applies to shared MD files with viewer permissions 👍